README.md

```diff
@@ -1,13 +1,58 @@
 # Springroll Seed
 
-Springroll Seed is intendend to be used as starting point for games projects. It comes bundled with Webpack, Babel, as well as a few other node modules to streamline development.
+SpringRoll Seed is intended to be used as a starting point for games projects. It comes bundled with Webpack, Babel, as well as a few other node modules to streamline development.
```

Comment:
"Developers should use SpringRoll Seed as a starting point for games projects.  It comes bundled with Webpack, Babel, and a few node modules to streamline development."